docs/operations-guide/migrating-from-h2.md

```diff
@@ -1,29 +1,56 @@
-# Migrating from using the H2 database to Postgres or MySQL/MariaDB
+# Migrating from the default H2 database to a production database
 
-If you decide to use the default application database (H2) when you initially start using Metabase, but later decide that you'd like to switch to a more production-ready database such as Postgres or MySQL/MariaDB, you're in the right place.
+- [Metabase's application database](#metabases-application-database)
+- [Databases we recommend for a storing your Metabase application data](#databases-we-recommend-for-storing-your-metabase-application-data)
+- [JAR: How to migrate from H2 to your production application database](#jar-how-to-migrate-from-h2-to-your-production-application-database)
+- [Docker: how to migrate from H2 to your production application database](#docker-how-to-migrate-from-h2-to-your-production-application-database)
+- [Troubleshooting migration issues](#troubleshooting-migration-issues)
 
-## Before you migrate
+## Metabase's application database
 
-- Avoid upgrading and migrating at the same time, since it can cause problems if one of the database schemas doesn't match.
-- You must be able to connect to the target Postgres or MySQL/MariaDB database in whatever environment you're running this migration command in. So, if you are attempting to move the data to a cloud database, make sure you can connect to that database.
+The difference between a local installation and a production installation of Metabase is the application database. The application database keeps track of all of your Metabase data: your questions, dashboards, collections, and so on.
 
-### Migrating when using Docker
+Metabase ships with an embedded H2 application database that is not recommended for running Metabase in production. The reason Metabase ships with the H2 database is because we want people to spin up Metabase on their local machine and start playing around with asking questions.
 
-We recommend running the migration outside of Docker. You'll need to copy the H2 file out of the Docker container before migrating. For example, if the container is called metabase, you'd run:
+If you want to run Metabase in production, you'll need to use a production-ready application database to store your application data. You can switch from using the default H2 application database at any time, but if you're planning on running Metabase in production, the sooner you migrate to a production application database, the better. If you keeping running Metabase with the default H2 application database, and you don't regularly back it up, the app db could get corrupted, and you could end up losing all of your questions, dashboards, collections, and other Metabase data.
 
-```
-docker cp metabase:/metabase.db/metabase.db.mv.db ./
-```
+You could also choose to run Metabase on a [Metabase Cloud](/pricing) plan, which takes care of all of this stuff for you.
```

Comment:
Perhaps link to the Cloud migration? https://www.metabase.com/cloud/docs/migrate/guide.html